flutter_gallery_sksl_warmup__transition_perf_e2e_ios32 flakes, flutter_01.sksl.json does not exist

Test is already marked flaky, so this isn't making the tree red.
```
flutter build ios --profile --bundle-sksl-path /Users/flutter/.cocoon/flutter/dev/integration_tests/flutter_gallery/flutter_01.sksl.json -t test_driver/transitions_perf_e2e.dart in /Users/flutter/.cocoon/flutter/dev/integration_tests/flutter_gallery
```
```
** BUILD FAILED **
/Users/flutter/.cocoon/flutter/dev/integration_tests/flutter_gallery/flutter_01.sksl.json does not exist.
```
[flutter_gallery_sksl_warmup__transition_perf_e2e_ios32_1a9ea39_1.log](https://github.com/flutter/flutter/files/5269866/flutter_gallery_sksl_warmup__transition_perf_e2e_ios32_1a9ea39_1.log)

--- a/dev/devicelab/lib/tasks/perf_tests.dart
+++ b/dev/devicelab/lib/tasks/perf_tests.dart
@@ -746,6 +746,7 @@ class PerfTestWithSkSL extends PerfTest {
         'run',
         '--verbose',
         '--verbose-system-logs',
+        '--purge-persistent-cache',
         '--profile',
         if (cacheSkSL) '--cache-sksl',
         '-d', _device.deviceId,

